prov/rxd/src/rxd.h

```diff
@@ -223,6 +224,8 @@ struct rxd_ctrl_hdr {
 	uint64_t tag;
 	uint32_t op;
 	uint32_t seg_size;
+	uint8_t source[RXD_NAME_LENGTH];
+	void *inline_data;
```

Comment:
Don't embed pointers into protocol messages.  The value isn't usable, and the size of the pointer will differ based on the platform.